Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -8,7 +8,7 @@ local QuestieDB = QuestieLoader:ImportModule("QuestieDB")
 ---@type QuestieQuestBlacklist
 local QuestieQuestBlacklist = QuestieLoader:ImportModule("QuestieQuestBlacklist")
 
-SeasonOfDiscovery.currentPhase = 1 -- TODO: Use API function which hopefully will come in the future
+SeasonOfDiscovery.currentPhase = 2 -- TODO: Use API function which hopefully will come in the future
```

Comment:
This is not required as it will be updated dynamically on init.

```suggestion
SeasonOfDiscovery.currentPhase = 1 -- TODO: Use API function which hopefully will come in the future
```

Reply:
Ah okay, thought it wasn't implemented yet so wanted to just bump it whilst that functionality is being worked on.

Reply:
This is the code for it https://github.com/Questie/Questie/blob/c51d80babf99f56fd1d654bdb5be3e2fc1e49428/Database/Corrections/SeasonOfDiscovery.lua#L948-L961